types/src/front/assistant/actions/code_interpreter.ts

```diff
@@ -0,0 +1,30 @@
+import { ModelId } from "../../../shared/model_id";
+import { BaseAction } from "../../lib/api/assistant/actions";
+
+export type CodeInterpreterRuntypeEnvironmentType = "js_with_react" | "python";
```

Comment:
We are just missing on type here which is "js"

Reply:
for javascript without React, which is the current implementation of the design doc.